ui/app/components/task-sub-row.hbs

```diff
@@ -1,16 +1,16 @@
-<tr class="task-sub-row"
+<tr class="task-sub-row {{if @active "is-active"}}"
 	{{keyboard-shortcut
 		enumerated=true
 		action=(action "gotoTask" this.task.allocation this.task)
 	}}
 >
 	<td colspan={{@namespan}}>
-		/
-		<LinkTo @route="allocations.allocation.task" @models={{array this.task.allocation this.task}}>
-			{{this.task.name}}
-		</LinkTo>
-		{{!-- TODO: in-page logs --}}
-		{{!-- <FlightIcon @name="logs" /> --}}
+		<div class="name-grid">
+			<LinkTo title={{this.task.name}} class="task-name" @route="allocations.allocation.task" @models={{array this.task.allocation this.task}}>{{this.task.name}}</LinkTo>
+			<button type="button" class="logs-sidebar-trigger button is-borderless is-inline is-compact" onclick={{action "handleTaskLogsClick" this.task}}>
+				<FlightIcon @name="logs" />View Logs
+			</button>
+		</div>
```

Comment:
`comment`: I think this is going to cause your test to fail. Maybe add a `data-test` attribute to your `LinkTo` component and assert on that selector instead of `.name-grid`.